samply/src/windows/profiler.rs

```diff
@@ -100,6 +100,10 @@ pub fn start_recording(
 
                 pids.push(child.id());
 
+                if process_launch_props.ignore_exit_code {
```

Comment:
I don't think this is right - you need to wait for the child process to finish before you launch the next iteration. Can you combine this check with the existing `!exit_status.success()` check?

Reply:
You are correct, this condition should go after the `wait` call. Have updated 👍